server/server.go

```diff
@@ -61,10 +65,35 @@ func New(cfg *config.Config) (*Server, error) {
 		})
 	})
 
+	// Serve static content
+	workDir, _ := os.Getwd()
+	filesDir := filepath.Join(workDir, "web")
+	FileServer(router, "/", http.Dir(filesDir))
```

Comment:
Shouldn't be serving out of the root, as this allows for collisions with `/photos` and doesn't programmatically separate out functionality

Please use a route like `/ui`

Reply:
I think the UI should be served from `/`. We will just have to be careful not to have routes that collide.

Reply:
I think the home page could be served from /, but resources should be under /ui. That way it's not a wildcard route, and they're still programmatically separated

Reply:
Resolved in bef0303.